package.json

```diff
@@ -27,7 +29,10 @@
     "lint": "npm-run-all lint:*",
     "lint:addon": "npm run package && addons-linter ./addon.xpi -o text",
     "lint:js": "eslint addon",
-    "build": "mkdir -p addon/build && cp $(node -e 'console.log(require.resolve(\"testpilot-ga\"))') addon/build/testpilot-ga.js && web-ext build --source-dir=addon --overwrite-dest",
+    "build": "npm-run-all build:*",
```

Comment:
The `build` script was getting too long for my eyes, so I split it into 3 sub-commands.

Reply:
👍